docs/html/installing.rst

```diff
@@ -6,7 +6,7 @@ Installation
 Do I need to install pip?
 -------------------------
 
-pip is already installed if you are using Python 2 >=2.7.9 or Python 3 >=3.4
+pip is already installed if you are using Python 2 >=2.7.9 or Python 3 >=3.5
```

Comment:
To me, putting 3.5 implies that pip was not included in lower versions. I think we should keep this as 3.4.